src/counts.jl

```diff
@@ -9,53 +9,90 @@
 typealias IntUnitRange{T<:Integer} UnitRange{T}
 
 #### functions for counting a single list of integers (1D)
+"""
+    addcounts!(r, x, levels[, wv])
 
+Fill the array `r` with the number of occurrences in `x` of
```

Comment:
Well, AFAICT `r[xi - b] += 1` is the only place where `r` is set, to it's adding, not overwriting.